### ISSUE TITLE
Support for Accessibility (WCAG 2.0AA)

### ISSUE THREAD

**Issue by asheshv** [ticlo/rc-dock#36]
Thanks for the rc-dock library.

I would like to add accessibility support [Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).

Would you accept it?

**Comment by rinick** [comment]
yes, the would be nice, is it just css and className changes?

**Comment by asheshv** [comment]
I am not sure, but - I would share the actual plan before proceeding.

— Thanks, Ashesh

**Comment by asheshv** [comment]
I think - we need both CSS, and javascript changes.
References:
- https://www.w3.org/TR/wai-aria-practices/#tabpanel
- https://www.w3.org/TR/wai-aria-practices/#windowsplitter (It is still in progress - no examples available yet)

These links also have some examples attached to it, which I will be referring during implementation.

To start with I will change HTML Element attributes like:
- [Role, Property, State, and Tabindex Attributes](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html#rps_label) for tabs & tabpanels.
- Javascript changes to support the keyboard shortcuts

**Comment by rinick** [comment]
@asheshv since rc-dock is based on rc-tabs, so there are already some aria attributes, please make you check that first.
also rc-tabs already supports keyboard shortcuts for switching tabs.

**Comment by asheshv** [comment]
> @asheshv since rc-dock is based on rc-tabs, there are already some aria attributes used there, please make sure you check that first.
Thanks for the pointer, it is missing the tabIndex attribute.
Hence - it does not allow to focus to panels. 
> also rc-tabs already supports keyboard shortcuts for switching tabs.
Thanks for the information.
